.header.md

```diff
@@ -1,20 +1,20 @@
-# AWS IP Address Manager Deployment Module
+# Terraform Module for Amazon VPC IP Address Manager on AWS
 
-This module can deploy simple or complex AWS IP Address Manager (IPAM) configurations. It is designed to be flexible for many different use cases. The most common use cases (IPAM designs) are highlighted in the [examples/](./examples/) directory. Below is a representation of a symmetrically nested, multi-region deployment that is possible; Its also possible to do [asymmetically nested deployments](images/asymmetrical_example.png) as well which we have as an [example](./examples/ipv4_basic).
+Built to accommodate a wide range of use cases, this Terraform module can deploy both simple and complex Amazon Virtual Private Cloud (Amazon VPC) IP Address Manager (IPAM) configurations. It supports both symmetrically nested, multi-Region deployments (most common IPAM designs) as well as [asymmetically nested deployments](images/asymmetrical_example.png).
 
-## Possible Symmetically Nested Pool Structure
+Refer to the [examples/](./examples/) directory in this GitHub repository for examples. 
 
-![symmetrically nested pool deployment](images/ipam_symmetrical.png "Region Separated Pools")
+This document describes a symmetrically nested pool structure, including its configuration, implementation details, examples for importing, requirements, and more. 
+## Architecture
 
-## Configuration via the `var.pool_configurations` variable
+![symmetrically nested pool deployment](images/ipam_symmetrical.png) "Region Separated Pools")
 
-This module leans heavily on the variable `var.pool_configurations` which is a multi-level nested map that describes exactly how you want your ipam pools to be nested. It can accept most `aws_vpc_ipam_pool` & `aws_vpc_ipam_pool_cidr` attributes (detailed below) as well as RAM share pools (at any level) to valid AWS principals. **Nested pools do not inherit attributes from their Source pool(s)** so all configuration options are available at each "level".
+## Configuration 
+This module strongly relies on the `var.pool_configuration` variable, which is a multi-level, nested map that describes how to nest your IPAM pools. It can accept most `aws_vpc_ipam_pool` and `aws_vpc_ipam_pool_cidr` attributes (detailed below) as well as RAM share pools (at any level) to valid AWS principals. Nested pools do not inherit attributes from their source pool(s), so all configuration options are available at each level.
```

Comment:
This module does allow for [inheriting locale](#local) to child pools to support Don't Repeat Yourself (DRY) concepts.

Reply:
Do you want me to add that line here?